File: SpringCloudDev/springcloud-vue-ui/src/entitys/integral/integralact.js
                                                                                            export default function area() {
  return {
    current: 1,
    size: 20,
    isNewRecord: false,
    query: {
    // 编号
  integralActId: undefined,
      // 积分活动编号
  actCode: undefined,
      // 积分活动名称
  actName: undefined,
      // 营销活动编号
  marketActId: undefined,
      // 营销活动结束时间
  endTime: undefined,
      // 所属部门
  department: undefined,
      // 积分类型
  integralType: undefined,
      // 积分有效期类型
  integralLimitTimeType: undefined,
      // 积分有效期年限
  integralLimitYearNum: undefined,
      // 积分到期月份
  integralEndMonth: undefined,
      // 计划积分
  prepareIntegralNum: undefined,
      // 计划人数
  prepareCount: undefined,
      // 活动描述
  actBreif: undefined,
      // 状态: audit_status_tosub：新增，audit_status_wait提交待审核，audit_status_pass审核通过，audit_status_nopass退回
  state: undefined,
      // 所属规则组
  ruleTeam: undefined,
      // 审核意见
  opinion: undefined,
      // 创建人
  crtUser: undefined,
      // 创建时间
  crtTime: undefined,
      // 最后更新人
  lstUpdUser: undefined,
      // 最后更新时间
  lstUpdTime: undefined,
      // 预留字段一
  reserveColumn1: undefined,
      // 预留字段二
  reserveColumn2: undefined,
      // 预留字段三
  reserveColumn3: undefined
          },
    rules: {
        integralActId: [{
  required: true,
  message: '请输入编号',
  trigger: 'blur'
  }],
          actCode: [{
  required: true,
  message: '请输入积分活动编号',
  trigger: 'blur'
  }],
          actName: [{
  required: true,
  message: '请输入积分活动名称',
  trigger: 'blur'
  }],
          marketActId: [{
  required: true,
  message: '请输入营销活动编号',
  trigger: 'blur'
  }],
          endTime: [{
  required: true,
  message: '请输入营销活动结束时间',
  trigger: 'blur'
  }],
          department: [{
  required: true,
  message: '请输入所属部门',
  trigger: 'blur'
  }],
          integralType: [{
  required: true,
  message: '请输入积分类型',
  trigger: 'blur'
  }],
          integralLimitTimeType: [{
  required: true,
  message: '请输入积分有效期类型',
  trigger: 'blur'
  }],
          integralLimitYearNum: [{
  required: true,
  message: '请输入积分有效期年限',
  trigger: 'blur'
  }],
          integralEndMonth: [{
  required: true,
  message: '请输入积分到期月份',
  trigger: 'blur'
  }],
          prepareIntegralNum: [{
  required: true,
  message: '请输入计划积分',
  trigger: 'blur'
  }],
          prepareCount: [{
  required: true,
  message: '请输入计划人数',
  trigger: 'blur'
  }],
          actBreif: [{
  required: true,
  message: '请输入活动描述',
  trigger: 'blur'
  }],
          state: [{
  required: true,
  message: '请输入状态: audit_status_tosub：新增，audit_status_wait提交待审核，audit_status_pass审核通过，audit_status_nopass退回',
  trigger: 'blur'
  }],
          ruleTeam: [{
  required: true,
  message: '请输入所属规则组',
  trigger: 'blur'
  }],
          opinion: [{
  required: true,
  message: '请输入审核意见',
  trigger: 'blur'
  }],
          crtUser: [{
  required: true,
  message: '请输入创建人',
  trigger: 'blur'
  }],
          crtTime: [{
  required: true,
  message: '请输入创建时间',
  trigger: 'blur'
  }],
          lstUpdUser: [{
  required: true,
  message: '请输入最后更新人',
  trigger: 'blur'
  }],
          lstUpdTime: [{
  required: true,
  message: '请输入最后更新时间',
  trigger: 'blur'
  }],
          reserveColumn1: [{
  required: true,
  message: '请输入预留字段一',
  trigger: 'blur'
  }],
          reserveColumn2: [{
  required: true,
  message: '请输入预留字段二',
  trigger: 'blur'
  }],
          reserveColumn3: [{
  required: true,
  message: '请输入预留字段三',
  trigger: 'blur'
  }]
      }
  }
}

File: SpringCloudDev/springcloud-business-api/springcloud-integral-api/src/main/java/org/springclouddev/integral/entity/IntegralAct.java
package org.springclouddev.integral.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;
import com.fasterxml.jackson.databind.ser.std.ToStringSerializer;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;
import org.springclouddev.core.mp.base.BaseEntity;

import java.time.LocalDateTime;

/**
 * 积分活动管理表实体类
 *
 * @author zhaobohao
 * @since 2020-01-15
 */
@Data
@Accessors(chain = true)
@TableName("tbl_integral_act")
@EqualsAndHashCode(callSuper = true)
@ApiModel(value = "IntegralAct对象", description = "积分活动管理表")
public class IntegralAct extends BaseEntity {

    private static final long serialVersionUID = 1L;

    /**
     * 编号
     */
    @ApiModelProperty(value = "编号")
    @TableId(value = "integral_act_id", type = IdType.NONE)
    @JsonSerialize(using = ToStringSerializer.class)
    private Long id;
    /**
     * 积分活动编号
     */
    @ApiModelProperty(value = "积分活动编号")
    @TableField("ACT_CODE")
    private String actCode;
    /**
     * 积分活动名称
     */
    @ApiModelProperty(value = "积分活动名称")
    @TableField("ACT_NAME")
    private String actName;
    /**
     * 营销活动编号
     */
    @ApiModelProperty(value = "营销活动编号")
    @TableField("MARKET_ACT_ID")
    private String marketActId;
    /**
     * 营销活动结束时间
     */
    @ApiModelProperty(value = "营销活动结束时间")
    private String endTime;
    /**
     * 所属部门
     */
    @ApiModelProperty(value = "所属部门")
    @TableField("DEPARTMENT")
    private String department;
    /**
     * 积分类型
     */
    @ApiModelProperty(value = "积分类型")
    @TableField("INTEGRAL_TYPE")
    private String integralType;
    /**
     * 积分有效期类型
     */
    @ApiModelProperty(value = "积分有效期类型")
    @TableField("INTEGRAL_LIMIT_TIME_TYPE")
    private String integralLimitTimeType;
    /**
     * 积分有效期年限
     */
    @ApiModelProperty(value = "积分有效期年限")
    @TableField("INTEGRAL_LIMIT_YEAR_NUM")
    private Long integralLimitYearNum;
    /**
     * 积分到期月份
     */
    @ApiModelProperty(value = "积分到期月份")
    @TableField("INTEGRAL_END_MONTH")
    private String integralEndMonth;
    /**
     * 计划积分
     */
    @ApiModelProperty(value = "计划积分")
    @TableField("PREPARE_INTEGRAL_NUM")
    private Long prepareIntegralNum;
    /**
     * 计划人数
     */
    @ApiModelProperty(value = "计划人数")
    @TableField("PREPARE_COUNT")
    private Long prepareCount;
    /**
     * 活动描述
     */
    @ApiModelProperty(value = "活动描述")
    @TableField("ACT_BREIF")
    private String actBreif;
    /**
     * 状态: audit_status_tosub：新增，audit_status_wait提交待审核，audit_status_pass审核通过，audit_status_nopass退回
     */
    @ApiModelProperty(value = "状态: audit_status_tosub：新增，audit_status_wait提交待审核，audit_status_pass审核通过，audit_status_nopass退回")
    @TableField("STATE")
    private String state;
    /**
     * 所属规则组
     */
    @ApiModelProperty(value = "所属规则组")
    @TableField("RULE_TEAM")
    private String ruleTeam;
    /**
     * 审核意见
     */
    @ApiModelProperty(value = "审核意见")
    @TableField("OPINION")
    private String opinion;
    /**
     * 预留字段一
     */
    @ApiModelProperty(value = "预留字段一")
    @TableField("RESERVE_COLUMN1")
    private String reserveColumn1;
    /**
     * 预留字段二
     */
    @ApiModelProperty(value = "预留字段二")
    @TableField("RESERVE_COLUMN2")
    private String reserveColumn2;
    /**
     * 预留字段三
     */
    @ApiModelProperty(value = "预留字段三")
    @TableField("RESERVE_COLUMN3")
    private String reserveColumn3;


}
